<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLevelsDock</class>
 <widget class="QDockWidget" name="QLevelsDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Levels</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QListWidget" name="levelList"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="levelUpButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="S2PEditor.qrc">
            <normaloff>:/Common/assets/images/GUI_Icons/arrowUp.png</normaloff>:/Common/assets/images/GUI_Icons/arrowUp.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="levelDownButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="S2PEditor.qrc">
            <normaloff>:/Common/assets/images/GUI_Icons/arrowDown.png</normaloff>:/Common/assets/images/GUI_Icons/arrowDown.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="loadButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="S2PEditor.qrc">
            <normaloff>:/Common/assets/images/GUI_Icons/saveLoad.png</normaloff>:/Common/assets/images/GUI_Icons/saveLoad.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteLevelButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="S2PEditor.qrc">
            <normaloff>:/QObjectsDock/assets/images/GUI_Icons/error.png</normaloff>:/QObjectsDock/assets/images/GUI_Icons/error.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="idLabel">
      <property name="text">
       <string>&lt;No item selected&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="sourceLine"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="S2PEditor.qrc"/>
 </resources>
 <connections/>
</ui>
